src/OrchardCore.Modules/OrchardCore.Facebook/Deployment/FacebookLoginDeploymentSource.cs

```diff
@@ -20,14 +21,12 @@ public async Task ProcessDeploymentStepAsync(DeploymentStep step, DeploymentPlan
 
             if (facebookLoginStep == null)
             {
-                return;
+                await Task.CompletedTask;
             }
 
-            var settings = await _facebookService.GetSettingsAsync();
-
             result.Steps.Add(new JObject(
                 new JProperty("name", "FacebookLogin"),
-                new JProperty("FacebookLogin", JObject.FromObject(settings))
+                new JProperty("FacebookLogin", JObject.FromObject(_facebookSettings))
```

Comment:
It depends on your configuration, one scenario if you configure AzureAD from within `appsettings.json` then export its settings you will get the values from the site settings instead which is confusing

In case if we go with this option, we might need to display an warning alert before export, so the user can knows what will be exported

Reply:
When you're exporting from an Orchard app, you're exporting the content and configuration of the app, not the environment. But besides the theoretical implications, this also has a practical security issue: it allows you to export potentially secret configuration from the hosting environment as a user.

Reply:
That's a good reason to use Secrets modules introduced by Dean time ago

Something might related, what will happen if we export email settings?

Reply:
Well, we'll have a Secrets module or not, sooner or later, but right now, this change opens up a security hole, so we shouldn't do it.